src/Components/TournamentDetails/helpers/__tests__/getListOfTournamentsFields.test.js

```diff
@@ -0,0 +1,62 @@
+
+import React from 'react';
+import {getListOfTournamentsFields} from "../getListOfTournamentsFields";
+import { renderToStaticMarkup } from 'react-dom/server';
+
+describe('getListOfTournamentsFields', () => {
+  it('выполнить return если tournament отсутствует(undefined)', () => {
```

Comment:
Тут лучше написать что-то вроде:
```suggestion
  it('Должен вернуть undefined для пустого объекта c турниром', () => {
```
т.к. return без указания значений отдает undefined